notifications.json

```diff
@@ -0,0 +1,861 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0",
+        "title": "Notification API",
+        "description": "Notification API"
+    },
+    "host": "e2effzjxvtgbmuilw-mock.stoplight-proxy.io",
+    "basePath": "/v1",
+    "schemes": [
+        "https"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/Messages/{id}": {
```

Comment:
the majority of API endpoints are lowercase, avoid mixing cases makes the endpoint more predictable